crates/nu-command/tests/commands/empty.rs

```diff
@@ -1,5 +1,42 @@
 use nu_test_support::{nu, pipeline};
 
+#[test]
+fn emptiness_for_nothing() {
+    let actual = nu!(
+        cwd: ".", pipeline(
+        r#"
+            $nothing | empty?
+        "#
+    ));
+
```

Comment:
Why the assertion here is that `empty?` for the input value $nothing is false?

Reply:
typo 😬. Was just running some tests for a bug.